docs/config/node.rst

```diff
@@ -98,10 +98,13 @@ Read-only node
   If set to ``true``, the node will only allow SQL statements which are
   resulting in read operations.
 
+Networking
```

Comment:
Re-structuring inspired by https://www.elastic.co/guide/en/elasticsearch/reference/7.10/modules-network.html

Added a new section gather all relevant settings (undocumented before TCP and transport settings goes there as well)